data/boarding missions.txt

```diff
@@ -24,6 +24,356 @@ mission "Assisting Merchant"
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Weed)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> along with lucrative offer for space weed. "Only 6000 Credits a Pound!"`
+				decline
+	destination Greenrock
+	
+mission "Assisting Syndicate"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment>.`
+				decline
+	destination Foundry
+
+mission "Assisting Syndicate (Protector)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> along with a message reading, "Trouble with Pirates? Hire a Syndicate Protector and your troubles will be over."`
+				decline
+	destination Foundry
+
+mission "Assisting Syndicate (Catalog)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> and the latest copy of the Syndicated Systems Catalog.`
+				decline
+	destination Foundry
+	
+mission "Assisting Syndicate (Dental)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> along with a message reading, "Work for the Syndicate! Our dental plans will keep you smiling!"`
+				decline
+	destination Foundry
+	
+mission "Assisting Oathkeepers"
+	assisting
+	repeat
+	to offer
+		random < 25
+	source
+		government "Navy (Oathkeeper)"
+	on offer
+		payment 7500
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Oathkeepers' cause, the captain gives you <payment>.`
+				decline
+	destination Farpoint
+
+mission "Assisting Navy"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment>.`
+				decline
+				
+mission "Assisting Navy (Weirdo)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment> and a message stating, "Next time we're on the same planet, stop by my berth and I'd be glad to give you a... private tour."`
```

Comment:
Pretty sure suggestive content won't merge.

Reply:
Reference to the naval officer from the FW story, quoted directly in fact

Reply:
I'd say toss this one entirely. It ruins the gimmick of the FW mission in question.